theme/icons/file-video.svg

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
The camcorder looks a little squashed, I think it could be a little better if the left side moved in a bit away from the edge